Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,7 @@
 function sum(elements) {
+  return elements
+    .filter((element) => typeof element === "number")
+    .reduce((acc, num) => acc + num, 0);
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
sum([Infinity, 1]);
sum([Infinity, Infinity]);
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```